sh/release

```diff
@@ -14,10 +14,22 @@ fi
 
 mkdir -p release
 
+# release binaries
 for plat in linux64 darwin
 do
   sh/cross urbit "$plat"
 
   echo "packaging release/urbit-$plat-$ver.tgz"
   (cd cross/$plat; tar cz .) > release/urbit-$plat-$ver.tgz
 done
+
+# release pill
+cp bin/brass.pill release/urbit-v$ver.pill
```

Comment:
You may want `nix-build brass` here instead -- the brass pill in `bin/` is not updated on any kind of regular basis.